for_developers/setup_guide.md

```diff
@@ -19,11 +26,17 @@ pip install -e .
 otx install -v
 ```
 
+In case of the error: `File "setup.py" not found. Directory cannot be installed in editable mode:`, uprade pip:
+
+```
+pip install --upgrade pip
+```
+
 ### With PIP & 'otx install'
 
 ```console
 # Create venv
-python -m venv venv
+python3.10 -m venv venv
```

Comment:
It's 3.9 already in the requirements above, so I think we shouldn't specify the exact version here

Reply:
Done, thank you